src/fe_cell_scalar_values.jl

```diff
@@ -71,18 +72,18 @@ function FECellValues{dim, T, FS <: FunctionSpace, GS <: FunctionSpace, shape <:
 
     detJdV = zeros(T, n_qpoints)
 
-    FECellValues(N, dNdx, dNdξ, detJdV, quad_rule, func_space, M, dMdξ, geom_space)
+    FECellScalarValues(N, dNdx, dNdξ, detJdV, quad_rule, func_space, M, dMdξ, geom_space)
 end
 
 
 """
-Updates the `FECellValues` object for an element.
+Updates the `FECellScalarValues` object for an element.
```

Comment:
maybe: "Updades the `AbstractFEValues` object for an element" since this method is used for them all?